src/main/java/org/craftercms/studio/api/v2/exception/content/FolderExistException.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (C) 2007-2022 Crafter Software Corporation. All Rights Reserved.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as published by
+ * the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package org.craftercms.studio.api.v2.exception.content;
+
+import org.craftercms.studio.api.v1.exception.ServiceLayerException;
+
+public class FolderExistException extends ServiceLayerException {
```

Comment:
@phuongnq any time a new exception is created it should be added to the exception handlers in API 2, if not required for now at least add a TODO here to remember it later

Reply:
Added to exception handlers in API 2. Thank you!